server/main.py

```diff
@@ -7,6 +7,23 @@
 from fastapi.encoders import jsonable_encoder
 import os
 import shutil
+import psycopg2
+import time
+
+
+def init_sql():
+    with conn.cursor() as cursor:
+        cursor.execute(
+            'CREATE TABLE applications (id INT PRIMARY KEY, name VARCHAR(64))')
+
+
+time.sleep(10)
```

Comment:
I suppose this was done to wait for db initilization? This also can be moved to docker-compose https://stackoverflow.com/questions/47615751/docker-compose-run-a-script-after-container-has-started